UnityProject/Assets/Scripts/Items/EncryptionKey.cs

```diff
@@ -34,8 +36,11 @@ public enum EncryptionKeyType
 /// <summary>
 ///     Encryption Key properties
 /// </summary>
-public class EncryptionKey : NetworkBehaviour
+public class EncryptionKey : NetworkBehaviour, ICheckedInteractable<HandApply>, ICheckedInteractable<MouseDrop>
```

Comment:
codacy is right, give these types a home

Reply:
im sorry i don't get what codacy wants?

Reply:
namespace. These poor types have no home to live in and wander in the global scope

Reply:
![image](https://user-images.githubusercontent.com/34368774/144340921-8a37cd17-0091-4eb9-8905-0b1f75446943.png)